org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/RefactorProposalUtility.java

```diff
@@ -844,4 +862,16 @@ public boolean isMoveAvaiable() {
 			return !supportedDestinationKinds.isEmpty();
 		}
 	}
+
+	public static class ExtractMethodInfo {
```

Comment:
This is duplicated with InferSelection class under file GetInferSelectionHandler.java.

Reply:
Will use InferSelection instead of this class.